uplink/src/base/actions/logging/logcat.rs

```diff
@@ -0,0 +1,184 @@
+use chrono::{Datelike, Local, Timelike};
```

Comment:
Logcat and journalctl have different behaviors. When we restart logcat, it prints all the logs with the new config again, while journalctl starts printing logs from the instant it was started. Journalctl can be modified to work like logcat by launching without `-f` pushing its output and then launching with `-f`. Logcat can be modifed to work like journalctl by ignoring all logs before the launch time.

Reply:
We'll let this slide? I think making any change in behavior such as either of these could lead to unexpected complexity and difficulties in maintenance.